packages/ai-rules/.ruler/data-modeling/dbt_model_development.md

```
@@ -0,0 +1,15 @@
+# Read the following docs in data-modeling repo
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)